File: UpmeetEventSystem/ClientApp/src/app/Event.ts
export class Event {
    eventId: number;
    eventName: string;
    eventDate: Date;
    eventLocation: string;
    isFavorite: boolean;

    constructor(eventId: number, eventName: string, eventDate: Date, eventLocation: string, isFavorite: boolean){
    this.eventId = eventId;
    this.eventName = eventName;
    this.eventDate = eventDate;
    this.eventLocation = eventLocation;
    this.isFavorite = isFavorite;
    }

}

File: UpmeetEventSystem/Models/Event.cs
using System;
using System.Collections.Generic;

namespace UpmeetEventSystem.Models
{
    public partial class Event
    {
        public Event()
        {
            Favorites = new HashSet<Favorite>();
        }

        public int EventId { get; set; }
        public string? EventName { get; set; }
        public DateTime? EventDate { get; set; }
        public string? EventLocation { get; set; }
        public bool? IsFavorite { get; set; }

        public virtual ICollection<Favorite> Favorites { get; set; }
    }
}
